src/sentry/static/sentry/app/views/alerts/details/activity/dateDivider.tsx

```diff
@@ -1,43 +1,31 @@
-/**
- * Activity component on Incident Details view
- * Allows user to leave a comment on an incident as well as
- * fetch and render existing activity items.
- */
-import React from 'react';
 import styled from '@emotion/styled';
 
 import space from 'app/styles/space';
 
-const DateDivider = styled(function DateDivider({children, ...props}) {
-  return (
-    <div {...props}>
-      <hr />
-      <TitleWrapper>
-        <Title>{children}</Title>
-      </TitleWrapper>
-    </div>
-  );
-})`
-  position: relative;
+const DateDivider = styled('div')`
   font-size: ${p => p.theme.fontSizeMedium};
-`;
-
-const Title = styled('span')`
-  background-color: ${p => p.theme.white};
-  padding: 0 ${space(2)};
-`;
-
-const TitleWrapper = styled('span')`
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
-
   display: flex;
   align-items: center;
   justify-content: center;
   color: ${p => p.theme.gray3};
+  margin: ${space(1.5)} 0;
+
+  &:before,
+  &:after {
+    content: '';
+    display: block;
+    flex-grow: 1;
+    height: 1px;
+    background-color: ${p => p.theme.borderLight};
+  }
+
+  &:before {
+    margin-right: ${space(2)};
+  }
+
+  &:after {
+    margin-left: ${space(2)};
+  }
```

Comment:
Simplifies this code a bit by using before and after pseudo elements to create the lines.